Update EU for eForms (#12)
open-contracting-extensions/ocds_eu_extension: README.md

Reviewer: line 5 only directs users to the TED guidance, it needs to refer users to the eforms ocds guidance too. Suggest changing to:

For complete guidance on meeting the disclosure requirement of European law, see [OCDS for European Union](https://standard.open-contracting.org/profiles/eu/master/en/) for correspondences to Tender Electronic Daily (TED) and [OCDS for eforms](https://standard.open-contracting.org/profiles/eforms/latest/en/) for correspondences to eforms.

Author: Good spot!

Implements fields and codes that are specific to European law.

> For complete guidance on meeting the disclosure requirements of European law, see [OCDS for eForms](https://standard.open-contracting.org/profiles/eforms/latest/en/) and [OCDS for European Union](https://standard.open-contracting.org/profiles/eu/latest/en/).

Reviewer 2: ```suggestion
For complete guidance on meeting the disclosure requirements of European law, see [OCDS for eForms](https://standard.open-contracting.org/profiles/eforms/latest/en/) for the 2019 regulation, or [OCDS for European Union](https://standard.open-contracting.org/profiles/eu/latest/en/) for the 2015 regulation.
```

Implements fields and codes that are specific to European law.

For complete guidance on meeting the disclosure requirements of European law, see [OCDS for eForms](https://standard.open-contracting.org/profiles/eforms/) and [OCDS for European Union](https://standard.open-contracting.org/profiles/eu/master/en/).

## Guidance

If items have at most one delivery address, use the [Location](https://extensions.open-contracting.org/en/extensions/location/) extension instead ([see discussion](https://github.com/open-contracting/ocds-extensions/issues/115)).

> ## Legal context

Reviewer 2: Maybe delete this section, because the EU extension exists to fill in gaps for eForms/TED that don't make sense to split out into other extensions (and it links to the profiles in the intro).

Author: Done in 382b0f56c80e342570a5bca220e543c7dfd2acc6

## Legal context

In the European Union, this extension's fields and codes correspond to the following eForms [business terms](https://docs.ted.europa.eu/eforms/latest/reference/business-terms/) and business groups:

> * eForms BT-99 (Review Deadline Description)

Reviewer: ```suggestion
* BT-99 (Review Deadline Description)
```
Not sure why this one had eforms in front of it when none of the others do

Author: Just a copy-paste error :-)

          "durationInDays": 30,
          "startDate": "2020-11-06T00:00:00Z",
          "endDate": "2020-12-06T00:00:00Z"
        },
        "additionalClassifications": [
          {
            "id": "oth-serv-contr",
            "scheme": "eu-cvd-contract-type",

>             "description": "other service contract"

Reviewer: ```suggestion
            "description": "Other service contract"
```

Being pedantic but in the authority table this is capitalised and guidance.yaml doesn't specify to lowercase the description prior to mapping it over to the ocds :)

Reviewer: It was me that did the guidance though and I put it in as lowercase so this is entirely my bad!
